distribution/docs/src/main/resources/content/_installing/high-availability-initial-setup.adoc

```diff
@@ -0,0 +1,31 @@
+:title: High Availability Initial Setup
+:type: installing
+:status: published
+:summary: Initial setup in a Highly Available Cluster.
+:project: ${branding}
+:order: 09
+
+=== {title}
+
+This section describes how to complete the initial setup of the ${branding} in a <<_introduction_to_high_availability,Highly Available Cluster>>.
+
+.Prerequisites
+* An HTTP proxy that can route HTTP traffic according to the pattern described in the Introduction to High Availability.
+The open-source proxy that was used for testing was HAProxy.
+(Refer to HAProxy's http://cbonte.github.io/haproxy-dconv/1.7/management.html#4[Management Guide] for setup instructions.)
+* Solr Cloud: See <<_solr_cloud_prerequisites, Solr Cloud section>> for guidance on installation as well
+ as configuring the ${branding}s to connect to Solr Cloud
```

Comment:
✏️  `See the Solr Cloud section for installation and configuration guidance to connect ${branding} nodes to Solr Cloud. `

Reply:
Fixed